channeldb/migration_01_to_11/graph.go

```diff
@@ -692,6 +692,8 @@ func fetchLightningNode(nodeBucket *bbolt.Bucket,
 	return deserializeLightningNode(nodeReader)
 }
 
+var globalFeatures = map[lnwire.FeatureBit]string{}
```

Comment:
this was the global features map at the time this migration happened. either way, it is only inspected if `HasFeature` is called, which doesn't happen during the migration. it could equally be nil and wouldn't affect serialization, perhaps that is better?

Reply:
Ok. With nil the migration would at least fail (?) if we would use it. Seems better.

Reply:
done.